projects/plugins/jetpack/modules/custom-post-types/portfolios.php

```diff
@@ -821,27 +967,33 @@ static function get_project_class( $portfolio_index_number, $columns ) {
 		 * @param string $class class name of the div.
 		 * @param int $portfolio_index_number iterator count the number of columns up starting from 0.
 		 * @param int $columns number of columns to display the content in.
-		 *
 		 */
-		return apply_filters( 'portfolio-project-post-class', implode( " ", $class ) , $portfolio_index_number, $columns );
+		return apply_filters(
+			'portfolio-project-post-class', // phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores
```

Comment:
It's too late to change it I'm afraid, since it may be used in the wild at this point. Is it worth deprecating it and replacing it with a better named filter? I'm open for it, let me know what you think.